rollup.config.mjs

```diff
@@ -15,13 +18,17 @@ export default {
     sourcemap: true,
     interop: false
   }],
-  external: [
-    'clsx',
-    'react',
-    'react-dom'
-  ],
+  external: id => !id.startsWith('.') && !isAbsolute(id),
   plugins: [
-    sourcemaps(),
-    nodeResolve()
+    babel({
+      babelHelpers: 'runtime',
+      skipPreflightCheck: true,
```

Comment:
Not sure if it is worth enabling it? Does it slow down the build by much?

Reply:
It builds in 1s in both cases, but I ran into issues before so I had to add this. I'll remove it now.